### DISCUSSION TITLE
Stop generating a response ?

### DISCUSSION THREAD

**Discussion by AnthonyBourassa** [llm-workflow-engine/llm-workflow-engine#100] [Q&A]
Excuse me, I'm new to this. Is there a way to make chatgpt stop generating a response if it get too long from the command line or with python ? Thank you.

**Comment by thehunmonkgroup** [comment]
You can send a `SIGUSR1` (non-Windows) to the running shell to stop text generation.

Windows theoretically will work with a `SIGBREAK`, but I have not tested it.
